config.json

```diff
@@ -992,6 +992,18 @@
         "text_formatting"
       ]
     },
+    {
+      "slug": "hangman",
+      "uuid": "ab3f8bf4-cfae-4f7a-b134-bb0fa4fafa63",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 6,
+      "topics": [
+        "strings",
+	"FRP"
```

Comment:
In the [topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt), we have reactive_programming and functional_programming, could you use those instead of "FRP", which is not on the list?